src/mutations/article/updateArticleSensitive.ts

```diff
@@ -0,0 +1,20 @@
+import { fromGlobalId } from 'common/utils'
+import { MutationToUpdateArticleSensitiveResolver } from 'definitions'
+
+const resolver: MutationToUpdateArticleSensitiveResolver = async (
+  _,
+  { input: { id, sensitive } },
+  { viewer, dataSources: { articleService, draftService } }
```

Comment:
`viewer` is not used？

Reply:
Yeah it's not in use since `@auth(mode: "${AUTH_MODE.admin}")` does the authentication, we can remove this line.